armi/reactor/converters/geometryConverters.py

```diff
@@ -1180,6 +1180,27 @@ class ThirdCoreHexToFullCoreChanger(GeometryChanger):
         geometry.DomainType.THIRD_CORE, geometry.BoundaryType.PERIODIC
     )
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.listOfVolIntegratedParamsToScale = []
+
+    def _scaleBlockVolIntegratedParams(self, b, direction):
+        if direction == "up":
+            op = operator.mul
+        elif direction == "down":
+            op = operator.truediv
+
+        for param in self.listOfVolIntegratedParamsToScale:
+            print(param)
+            print(b.p[param])
```

Comment:
Was this left intentionally? It's creating a ton of output.

Reply:
@keckler @john-science

Reply:
Haha, whoops!

Reply:
So much for code review...

I'll fix it.

Reply:
https://github.com/terrapower/armi/pull/753